software/pixhawk-connect.md

```diff
@@ -0,0 +1,20 @@
+# OBC Serial Connection to Pixhawk setup
+
+The OBC (a raspberry pi) can be configured to communicate with the Pixhawk over a serial connection. This allows the OBC to retrieve telemetry data from the pixhawk.
+
+## Setting up the Pi
+
+[This ardupilot wiki page](http://ardupilot.org/dev/docs/raspberry-pi-via-mavlink.html) details how to set up a serial connection between the Raspberry Pi and Pixhawk.
+
+### Installing dependencies for OBC
+
+Before starting make sure you have pip installed and are in a Python virtual environement.
+
+`apt-get install python-dev`
+`pip install future`
+`pip install pymavlink`
+`pip install dronekit`
```

Comment:
I think that most if not all of these packages exist in the `requirments.txt` file. Do we still need to install them separately?

Reply:
You're correct, I've removed the extra pip install commands. I've left the `python-dev` installation as this is not done by the `requirements.txt` file.